objects/src/accounts/delta.rs

```diff
@@ -0,0 +1,45 @@
+use super::{AccountStorage, Felt};
+use assembly::ast::ModuleAst;
+use crypto::merkle::{MerkleStoreDelta, MerkleTreeDelta};
+
+// ACCOUNT DELTA
+// ================================================================================================
+
+/// [AccountDelta] stores the differences between the initial and final account states.
+///
+/// The differences are represented as follows:
+/// - code: an Option<ModuleAst> that contains the updated code of the account.
+/// - nonce: if the nonce of the account has changed, the new nonce is stored here.
+/// - storage: an [AccountStorageDelta] that contains the changes to the account storage.
+/// - vault: an [MerkleTreeDelta] object that contains the changes to the account vault assets tree.
+#[derive(Debug, Clone)]
+pub struct AccountDelta {
+    pub code: Option<ModuleAst>,
```

Comment:
As mentioned in a couple of previous comments: maybe we should simplify and not include tracking of account code changes for now.